modules/post/multi/gather/apple_ios_backup.rb

```diff
@@ -3,6 +3,7 @@
 # Current source: https://github.com/rapid7/metasploit-framework
 ##
 
+require 'English'
```

Comment:
where did this come from?

Reply:
Probably from here: https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/SpecialGlobalVars

Reply:
Looks like the same change has been made to a bunch of files due to use of `raise $!` (replaced with `raise $ERROR_INFO`).

Prior to this PR:

```
$ grep -rn 'raise $!' modules/
modules/exploits/unix/webapp/php_include.rb:157:        raise $!
modules/exploits/linux/snmp/awind_snmp_exec.rb:92:      raise $!
modules/exploits/linux/snmp/awind_snmp_exec.rb:118:      raise $!
modules/exploits/linux/snmp/awind_snmp_exec.rb:143:      raise $!
modules/exploits/linux/samba/trans2open.rb:147:      raise $!
modules/exploits/multi/http/php_cgi_arg_injection.rb:133:      raise $!
modules/post/windows/gather/hashdump.rb:118:      raise $!
modules/post/windows/gather/cachedump.rb:398:    raise $!
modules/post/windows/manage/mssql_local_auth_bypass.rb:98:        raise $!
modules/post/windows/manage/mssql_local_auth_bypass.rb:114:        raise $!
modules/post/windows/manage/hashcarve.rb:82:      raise $!
modules/post/windows/manage/inject_host.rb:62:      raise $! unless e.result == 2
modules/post/multi/gather/apple_ios_backup.rb:248:          raise $!
modules/auxiliary/admin/serverprotect/file.rb:391:      raise $!
modules/auxiliary/admin/natpmp/natpmp_map.rb:111:      raise $!
modules/auxiliary/admin/oracle/sid_brute.rb:48:        raise $!
modules/auxiliary/admin/scada/multi_cip_command.rb:70:      raise $!
modules/auxiliary/admin/scada/multi_cip_command.rb:104:      raise $!
modules/auxiliary/gather/windows_deployment_services_shares.rb:156:      raise $!
modules/auxiliary/spoof/dns/bailiwicked_host.rb:223:      raise $!
modules/auxiliary/spoof/dns/bailiwicked_host.rb:378:          raise $!
modules/auxiliary/spoof/dns/bailiwicked_domain.rb:226:      raise $!
modules/auxiliary/spoof/dns/bailiwicked_domain.rb:379:          raise $!
modules/auxiliary/spoof/nbns/nbns_response.rb:159:        raise $!
modules/auxiliary/server/browser_autopwn.rb:985:            raise $!
modules/auxiliary/server/openssl_heartbeat_client_memory.rb:273:        raise $!
modules/auxiliary/server/netbios_spoof_nat.rb:99:      raise $!
modules/auxiliary/server/capture/sip.rb:228:      raise $!
modules/auxiliary/fuzzers/http/http_get_uri_long.rb:60:        raise $!
modules/auxiliary/fuzzers/http/http_get_uri_strings.rb:57:        raise $!
modules/auxiliary/fuzzers/ssh/ssh_kexinit_corrupt.rb:62:        raise $!
modules/auxiliary/fuzzers/ssh/ssh_version_corrupt.rb:59:        raise $!
modules/auxiliary/fuzzers/ssh/ssh_version_2.rb:55:        raise $!
modules/auxiliary/fuzzers/ssh/ssh_version_15.rb:55:        raise $!
modules/auxiliary/fuzzers/tds/tds_login_username.rb:134:        raise $!
modules/auxiliary/fuzzers/tds/tds_login_corrupt.rb:137:        raise $!
modules/auxiliary/fuzzers/smb/smb_create_pipe.rb:48:        raise $!
modules/auxiliary/fuzzers/smb/smb_ntlm1_login_corrupt.rb:61:        raise $!
modules/auxiliary/fuzzers/smb/smb_negotiate_corrupt.rb:57:        raise $!
modules/auxiliary/fuzzers/smb/smb_create_pipe_corrupt.rb:67:        raise $!
modules/auxiliary/fuzzers/smb/smb_tree_connect_corrupt.rb:72:        raise $!
modules/auxiliary/fuzzers/smb/smb_tree_connect.rb:54:        raise $!
modules/auxiliary/fuzzers/smb/smb2_negotiate_corrupt.rb:58:        raise $!
modules/auxiliary/fuzzers/smtp/smtp_fuzzer.rb:133:        raise $!
modules/auxiliary/dos/windows/ftp/iis_list_exhaustion.rb:68:      raise $!
modules/auxiliary/dos/windows/smb/vista_negotiate_stop.rb:59:        raise $!
modules/auxiliary/dos/misc/dopewars.rb:49:      raise $!
modules/auxiliary/scanner/dcerpc/management.rb:80:      raise $!
modules/auxiliary/scanner/dcerpc/windows_deployment_services.rb:52:      raise $!
modules/auxiliary/scanner/dcerpc/endpoint_mapper.rb:72:      raise $!
modules/auxiliary/scanner/dcerpc/tcp_dcerpc_auditor.rb:272:            raise $!
modules/auxiliary/scanner/dcerpc/tcp_dcerpc_auditor.rb:290:          raise $!
modules/auxiliary/scanner/ftp/ftp_version.rb:40:      raise $!
modules/auxiliary/scanner/vxworks/wdbrpc_bootline.rb:58:          raise $!
modules/auxiliary/scanner/vxworks/wdbrpc_bootline.rb:89:      raise $!
modules/auxiliary/scanner/vxworks/wdbrpc_version.rb:56:          raise $!
modules/auxiliary/scanner/vxworks/wdbrpc_version.rb:87:      raise $!
modules/auxiliary/scanner/afp/afp_server_info.rb:36:      raise $!
modules/auxiliary/scanner/afp/afp_server_info.rb:39:      raise $!
modules/auxiliary/scanner/snmp/snmp_enum.rb:871:      raise $!
modules/auxiliary/scanner/snmp/xerox_workcentre_enumusers.rb:54:      raise $!
modules/auxiliary/scanner/snmp/cisco_upload_file.rb:172:      raise $!
modules/auxiliary/scanner/snmp/ubee_ddw3611.rb:150:       raise $!
modules/auxiliary/scanner/snmp/cisco_config_tftp.rb:148:      raise $!
modules/auxiliary/scanner/snmp/epmp1000_snmp_loot.rb:159:      raise $!
modules/auxiliary/scanner/snmp/brocade_enumhash.rb:65:       raise $!
modules/auxiliary/scanner/snmp/snmp_enumshares.rb:56:      raise $!
modules/auxiliary/scanner/snmp/sbg6580_enum.rb:217:      raise $!
modules/auxiliary/scanner/snmp/snmp_enum_hp_laserjet.rb:142:    raise $!
modules/auxiliary/scanner/snmp/netopia_enum.rb:93:       raise $!
modules/auxiliary/scanner/snmp/aix_version.rb:65:      raise $!
modules/auxiliary/scanner/snmp/snmp_set.rb:84:      raise $!
modules/auxiliary/scanner/snmp/cnpilot_r_snmp_loot.rb:135:      raise $!
modules/auxiliary/scanner/telephony/wardial.rb:344:      raise $!
modules/auxiliary/scanner/msf/msf_rpc_login.rb:44:      raise $!
modules/auxiliary/scanner/sip/enumerator.rb:76:            raise $!
modules/auxiliary/scanner/sip/enumerator.rb:96:      raise $!
modules/auxiliary/scanner/sip/enumerator_tcp.rb:56:          raise $!
modules/auxiliary/scanner/sip/enumerator_tcp.rb:63:      raise $!
modules/auxiliary/scanner/backdoor/energizer_duo_detect.rb:115:      raise $!
modules/auxiliary/scanner/sap/sap_router_portscanner.rb:381:            raise $!
modules/auxiliary/scanner/sap/sap_service_discovery.rb:242:            raise $!
modules/auxiliary/scanner/portscan/tcp.rb:88:            raise $!
modules/auxiliary/scanner/discovery/udp_probe.rb:88:          raise $!
modules/auxiliary/scanner/discovery/udp_probe.rb:97:      raise $!
modules/auxiliary/scanner/h323/h323_version.rb:109:      raise $!
modules/auxiliary/scanner/vmware/vmauthd_version.rb:76:      raise $!
modules/auxiliary/scanner/natpmp/natpmp_portscan.rb:56:      raise $!
modules/auxiliary/scanner/telnet/lantronix_telnet_password.rb:62:      raise $!
modules/auxiliary/scanner/mysql/mysql_authbypass_hashdump.rb:65:      raise $!
modules/auxiliary/scanner/smb/smb_ms17_010.rb:167:      raise $!
modules/auxiliary/scanner/smb/smb_enumusers.rb:329:      raise $!
modules/auxiliary/scanner/smb/smb_lookupsid.rb:290:      raise $!
modules/auxiliary/scanner/misc/ibm_mq_enum.rb:173:      raise $!
modules/auxiliary/scanner/misc/poisonivy_control_scanner.rb:74:            raise $!
modules/auxiliary/scanner/misc/cctv_dvr_login.rb:68:      raise $!
modules/auxiliary/scanner/misc/oki_scanner.rb:131:      raise $!
modules/auxiliary/scanner/misc/rosewill_rxs3211_passwords.rb:63:      raise $!
modules/auxiliary/scanner/finger/finger_users.rb:135:      raise $!
modules/auxiliary/scanner/rogue/rogue_recv.rb:47:      raise $!
modules/auxiliary/scanner/ipmi/ipmi_dumphashes.rb:333:      raise $!
```

Reply:
In most cases we would probably circle back and replace `$!` and `$ERROR_INFO` with the more understandable

```
rescue => e
  raise e
```

Or just `raise` works too for most of these snippets, which will re-raise the caught exception